docs/crashlytics/overview.mdx

```diff
@@ -33,29 +33,28 @@ $ flutter pub get
 
 ### 3. Platform integration
 
-If you are migrating from a previous version of Crashlytics that used Fabric, please follow this [upgrade guide](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android).
+If you are migrating from a previous version of Crashlytics that used Fabric, please follow these
+guides and remove any legacy Fabric integration steps from your project:
+
+ - [Android](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android).
+ - [iOS/macOS](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=ios).
 
 #### a. Android
 
-1. Add the following classpaths to the `/android/build.gradle` file.
+1. Add the following classpaths to your `android/build.gradle` file.
 
-```dart
+```groovy {3} title="android/build.gradle"
 dependencies {
   // ... other dependencies
   classpath 'com.google.firebase:firebase-crashlytics-gradle:2.2.0'
 }
 ```
 
-2. Apply the following plugins to the `/android/app/build.gradle` file.
+2. Apply the following to the bottom of your `android/app/build.gradle` file.
 
-```dart
+```groovy {5} title="android/app/build.gradle"
 dependencies {
-  // ... other dependencies
-  // Optional but recommended: Add the Firebase SDK for Google Analytics.
-  implementation 'com.google.firebase:firebase-analytics:17.5.0'
-
-  // Add the Firebase Crashlytics SDK.
-  implementation 'com.google.firebase:firebase-crashlytics:17.2.1'
```

Comment:
I think you removed a bit too much here.

Reply:
These dependencies are managed by the plugin itself which includes them transitively, so its not required by the user to add this in